src/option.cpp

```diff
@@ -1430,6 +1430,13 @@ void register_options(void)
                   "0 = No change.");
    unc_add_option("nl_property_brace", UO_nl_property_brace, AT_IARF,
                   "Add or remove newline between C# property and the '{'.");
+   unc_add_option("nl_inside_namespace", UO_nl_inside_namespace, AT_UNUM,
+                  "The number of newlines after '{' of a namespace. This also "
+                  "adds newlines\nbefore the matching '}'.\n\n"
+                  "0 = Apply eat_blanks_after_open_brace or "
+                  "eat_blanks_before_close_brace if\n    applicable, otherwise "
```

Comment:
were `eat_blanks_after_open_brace` and `eat_blanks_before_close_brace` enabled by default?
making `nl_inside_namespace=0` will implicitly change the default behaviour which probably is to ignore.

Reply:
> were `eat_blanks_after_open_brace` and `eat_blanks_before_close_brace` enabled by default?

No.

> making `nl_inside_namespace=0` will implicitly change the default behaviour which probably is to ignore

It won't (at least it shouldn't, unless I messed something up; if you can show that I'm wrong, please do so!); if `nl_inside_namespace=0` there is no behavior change. That is, blanks "inside namespaces" are either ignored, or may be affected by `eat_blanks_after_open_brace` and `eat_blanks_before_close_brace` as before.

Is the confusion because you are interpreting the above to mean that eating blanks is *always* done if `nl_inside_namespace=0`? This sounds suspiciously related to the question I posed; would it be better to just say "0 = Ignore"? (But that might seem to imply that `eat_blanks_*_brace` *won't* be applied...)

Which is less confusing? Maybe there is a third option I haven't thought of? Suggestions for improvements always welcome!

Maybe "0 = Use \<options\> where applicable."; "use" instead of "apply" might be more clear that we aren't applying the *effect* of the options *as if* they were enabled, but we are applying the options as directed by their value (which might be 'do nothing')?